### PR TITLE
Update documentation to reflect the modified targets for building driver

### PR DESCRIPTION
**Pull request by nakulkorde**
The targets to build tarball, rpm and debian packages for driver were modified.
This PR addresses those changes in the documentation.

### CODE REVIEW

File: doc/src/fpga_api/quick_start/readme.md [hunk 1]
```diff
@@ -31,42 +31,65 @@ accelerator RTL with OPAE applications.
...
+distributions using RPM (e.g. Redhat, Fedora, Centos) and DEB (eg. Ubuntu) package managers.
+Download these package from the respective [release page on
+GitHub](https://github.com/OPAE/opae-sdk/releases) - they are named
+`opae-intel-fpga-driver-x.y.z-r.x86_64.rpm` and `opae-intel-fpga-driver_x.y.z-r_amd64.deb`, with `x.y.z-r` being the respective OPAE
 release's version number.
 
 ```eval_rst
 .. note::
-    The RPM package requires that the DKMS (Dynamic Kernel Module System)
+    The packages requires that the DKMS (Dynamic Kernel Module System)
```
Comment (luebbers) [inline doc/src/fpga_api/quick_start/readme.md]: "The packages *require* that..."

File: doc/src/fpga_api/quick_start/readme.md [hunk 2]
```diff
@@ -31,42 +31,65 @@ accelerator RTL with OPAE applications.
...
     package, version greater than 2.2, is already installed.
 ```
 
-For Redhat and Centos:
+For RPM base distributions:
```
Comment (luebbers) [inline doc/src/fpga_api/quick_start/readme.md]: "... RPM-based ..."

File: doc/src/fpga_api/quick_start/readme.md [hunk 3]
```diff
@@ -31,42 +31,65 @@ accelerator RTL with OPAE applications.
...
+For rpm packages:
+
 ```console
-$ sudo yum install opae-<release>-1.x86_64-libs.rpm
-$ sudo yum install opae-<release>-1.x86_64-tools.rpm
-$ sudo yum install opae-<release>-1.x86_64-devel.rpm
+$ sudo yum install opae-libs-<release>-1.x86_64.rpm
+$ sudo yum install opae-tools-<release>-1.x86_64.rpm
+$ sudo yum install opae-tools-extra-<release>-1.x86_64.rpm
+$ sudo yum install opae-devel-<release>-1.x86_64.rpm
```
Comment (luebbers) [inline doc/src/fpga_api/quick_start/readme.md]: Can we be sure that people will always deal with the `-1` version of the packages? Wouldn't that change if we had multiple release candidates?

File: doc/src/install_guide/installation_guide.md [hunk 4]
```diff
@@ -56,20 +56,30 @@ For building libopae-c, tools and samples, the following dependences are require
 * boost.x86\_64:           (tested with 1.53.0-26.el7)
 * boost-devel.x86\_64:     (tested with 1.53.0-26.el7)
 
-## Driver installation with DKMS rpm package ##
+## Driver installation with DKMS rpm and deb package ##
```
Comment (luebbers) [inline doc/src/install_guide/installation_guide.md]: Either "rpm and deb *packages*" or "rpm *or* deb package".

File: doc/src/install_guide/installation_guide.md [hunk 5]
```diff
@@ -56,20 +56,30 @@ For building libopae-c, tools and samples, the following dependences are require
...
 
-Install:
+Install rpm package:
 ```console
-$ sudo yum install opae-intel-fpga-drv-<release>.rpm
+$ sudo yum install opae-intel-fpga-driver-<release>.x86_64.rpm
 ```
 
-Uninsall:
+Uninsall rpm package:
```
Comment (luebbers) [inline doc/src/install_guide/installation_guide.md]: "Uninstall..." (carried over from previous commits).

File: doc/src/fpga_api/quick_start/readme.md (hunk 3 above)
Comment (nakulkorde) [reply doc/src/fpga_api/quick_start/readme.md]: I'll leave it as sudo yum install opae-libs-<release>.x86_64.rpm as I have done in other places.

**Pull request merged**